packages/typespec-python/test/azure/mock_api_tests/asynctests/test_azure_arm_models_common_types_managed_identity_asyncy.py

```diff
@@ -0,0 +1,63 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
the file name `test_azure_arm_models_common_types_managed_identity_asyncy.py` has surplus `y` before `.py`

Reply:
sorry for the carelessness, updated